apps/gallery/style/gallery.css

```diff
@@ -257,6 +257,11 @@ body.pickView #thumbnails {
   outline-offset: -0.2rem;
 }
 
+/* RTL modifications */
+html[dir=rtl] #thumbnails-number-selected {
+  direction: rtl;
+}
+
```

Comment:
I disagree; I think it's better to be able to see all the different states a single element can be in in one place. Otherwise it's easy for someone new to not know that there are overrides in other parts of the code base.